demo/DemoArticles/ArticleRef.js

```diff
@@ -0,0 +1,108 @@
+
+/**
+ * Type representing article source endpoints.
+ * @enum {ArticleRefSourceType}
+ */
+const ArticleRefSourceType = {
+  mobileView: 1,
```

Comment:
I know this is an enum pattern but I think you'll find your filenames more readable if you use a string here or convert from enum to string fileName().